_posts/2015-09-04-channel.html

```diff
@@ -3189,6 +3189,296 @@
 			</tr>
 		</table>
 
+
+
+		<h3>Video chapters</h3>
+
+		<h4>Get video chapters</h4>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">GET</td>
+				<td><a>https://api.ustream.tv/videos/VIDEO_ID/chapters.FORMAT</a> (format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Response</h6>
+		<p>An set of key-value structures under a "chapters" element.</p>
+		<p>Example:</p>
+		<pre><code class="json">"chapters": [
+        {
+            "id": 1,
+            "title": "First chapter",
+            "position": 10
+        },
+        {
+            "id": 2,
+            "title": "Second chapter",
+            "position": 60
+        }
+]</code></pre>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>unauthorized</td>
+				<td>401 Authorization required</td>
+				<td>The video is protected, only the owner with a valid token can access it.</td>
+			</tr>
+			<tr>
+				<td>lack_of_ownership</td>
+				<td>403 Forbidden</td>
+				<td>The given video is private or the channel is protected and the video does not belong to the user.</td>
```

Comment:
"The given video does not belong ..." talán könnyebben érthető, még ha nem is ennyire pontos.